void_migration/plotter.py

```diff
@@ -242,7 +238,7 @@ def plot_s(x, y, s, p, t, *args):
     plt.subplots_adjust(left=0, right=1, bottom=0, top=1)
     if hasattr(p, "plot_colorbar"):
         plt.colorbar(shrink=0.8, location="top", pad=0.01, ticks=ticks)
-    plt.savefig(p.folderName + "s_" + str(t).zfill(6) + ".png")
+    plt.savefig(p.folderName + "s_" + str(t).zfill(6) + ".png", bbox_inches="tight", dpi=100)
```

Comment:
please never use `bbox_inches="tight"`. If you want extra space for the colorbar, maybe we need a better way to implement this? Currently this breaks the existing plots, which are designed for minimal file size and to store just the information from the simulation. If you want production-ready graphs or videos, maybe they can be produced later in a second pass using the saved data?

Reply:
done